scripts/bert/finetune_classifier.py

```diff
@@ -89,89 +142,140 @@
 
 ctx = mx.cpu() if not args.gpu else mx.gpu()
 
+task = tasks[task_name]
+
 # model and loss
 dataset = 'book_corpus_wiki_en_uncased'
-bert, vocabulary = bert_12_768_12(dataset_name=dataset,
-                                  pretrained=True, ctx=ctx, use_pooler=True,
-                                  use_decoder=False, use_classifier=False)
-model = BERTClassifier(bert, dropout=0.1)
-model.classifier.initialize(init=mx.init.Normal(0.02), ctx=ctx)
-model.hybridize(static_alloc=True)
+bert, vocabulary = bert_12_768_12(
+    dataset_name=dataset,
+    pretrained=True,
+    ctx=ctx,
+    use_pooler=True,
+    use_decoder=False,
+    use_classifier=False)
 
```

Comment:
We can add a  parameter， bert_model_name,  for the pretrained bert model and then use 
```python
bert, vocabulary = nlp.model.get_model(bert_model_name, ....)
``` 
This allows the user to select which bert model to fine tune. This can be more flexible.